.eslintrc.js

```diff
@@ -4,7 +4,11 @@ module.exports = {
     browser: true,
     es2021: true,
   },
-  extends: ['eslint:recommended', 'plugin:vue/vue3-strongly-recommended'],
+  extends: [
+    'eslint:recommended',
+    'plugin:vue/vue3-strongly-recommended',
```

Comment:
No, we should be using `plugin:vue/strongly-recommended`. I didn't change that just because it's not the scope of this PR. If you disagree, let me know and I add it.

It is working, indeed. `plugin:vue/vue3-strongly-recommended` seems to cover the two versions, although [the documentation indicates it's aimed only for vue3](https://eslint.vuejs.org/rules/#priority-b-strongly-recommended-improving-readability-for-vue-js-3-x).

Reply:
Yeah. Maybe the style-guide is the same for both versions and the vue3 one just adds the new typescript styles.